OrbitLinuxTracing/PerfEvent.h

```diff
@@ -307,6 +307,9 @@ class UretprobesPerfEvent : public PerfEvent, public AbstractUprobesPerfEvent {
   pid_t GetPid() const { return ring_buffer_record.sample_id.pid; }
   pid_t GetTid() const { return ring_buffer_record.sample_id.tid; }
 
+  // Get AX register which typically holds return value.
```

Comment:
Add link to abi? If I recall correctly it always holds return value if there is one that is.